src/test.rs

```diff
@@ -88,7 +89,7 @@ pub fn this_host_triple() -> String {
         // For windows, this host may be different to the target: we may be
         // building with i686 toolchain, but on an x86_64 host, so run the
         // actual detection logic and trust it.
-        let tp = Box::new(currentprocess::TestProcess::default());
+        let tp: Box<TestProcess> = Box::default();
```

Comment:
I definitely don't like this one. The former spelling is much more what we'd see in other code that doesn't have the special case of being defaultable, and consistency helps read codebases.